data/udisks2.service.in

```diff
@@ -3,10 +3,27 @@ Description=Disk Manager
 Documentation=man:udisks(8)
 
 [Service]
-Type=dbus
 BusName=org.freedesktop.UDisks2
+CapabilityBoundingSet=CAP_SYS_RAWIO
 ExecStart=@udisksdprivdir@/udisksd
+IPAddressDeny=any
 KillSignal=SIGINT
+LimitMEMLOCK=0
+LockPersonality=yes
+MemoryDenyWriteExecute=yes
+NoNewPrivileges=yes
```

Comment:
Possibly used from libraries, please remove.

Reply:
Please note that `NoNewPrivileges=yes` is implied by many settings which use seccomp internally, for example `LockPersonality`, `MemoryDenyWriteExecute`, `SystemCallArchitectures` and
`SystemCallFilter`. `NoNewPrivileges=yes` in general blocks setuid helpers, do you think block device libraries really use them?

Otherwise none of the seccomp features can be used at all.